core/store/models/service_agreement.go

```diff
@@ -37,6 +54,20 @@ type ServiceAgreement struct {
 	JobSpecID     *ID         `json:"jobSpecId"`
 }
 
+// ServiceAgreementRequest encodes external ServiceAgreement json representation.
+type ServiceAgreementRequest struct {
+	Initiators             []InitiatorRequest     `json:"initiators"`
```

Comment:
Isnt it pragmatic Go to comment anything exported? We should do that here if so

Reply:
I believe that it is pragmatic to add comments(/docs via godoc) to public methods and types, but not fields of a type. It may be a little inconsistent that fields on a type aren't expected to be documented, but it seems pretty well established. [From the Golang blog](https://blog.golang.org/godoc-documenting-go-code):
> The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line.